f/briefing/f_orbatNotes.sqf

```diff
@@ -150,6 +150,19 @@ if (count _veharray > 0) then {
 
 };
 
+// Add button to take control of group when Arma does not update leadership correctly.
+// TO DO find better spot for this and/or don't show until after mission launch.
```

Comment:
Would at the bottom of the `Administration` tab be a better place for this? It generally has more technical info in it (mechanics, etc.) and would seem an okay place to put an action.